scripts/imageup

```diff
@@ -27,6 +27,12 @@ if [[ -z "$COMPOSE_ENV_FILE" || -z "$ENVIRONMENT" ]]; then
   exit 1
 fi
 
+# Skip automatic daily resets in production (we use Pegasus)
+if [[ "$1" == "--auto" && "$ENVIRONMENT" == "production" ]]; then
```

Comment:
suggest using an actual flag tool like we do in imageup elsewhere, since if someone does `imageup -k --auto, this will fail...`